### PR TITLE
Fix ricty diminished

### PR DESCRIPTION
**Pull request by neronplex**
updated font version.
fix font license.
use version stanza.

### CODE REVIEW

File: Casks/font-ricty-diminished.rb [hunk 1]
```diff
@@ -1,17 +1,17 @@
 cask :v1 => 'font-ricty-diminished' do
-  version '3.2.4'
-  sha256 '028f2b40faed8417fcc80a7b51592e9115a2c5af901aced833032df2983fdcf4'
+  version '4.0.1'
+  sha256 '08ab33a200778ee66af5618f34091dfe12451c675508cd01c8ede753bf3d6159'
 
-  url 'https://github.com/yascentur/RictyDiminished/archive/3.2.4.zip'
+  url 'https://github.com/yascentur/RictyDiminished/archive/#{version}.tar.gz'
```
Comment (vitorgalvao) [inline Casks/font-ricty-diminished.rb]: Needs to use double quotes, or it won’t work.

**Comment by vitorgalvao** [comment]
There are merge conflicts that need fixing.

**Comment by neronplex** [comment]
Once you close because I did not know the way of conflict resolution. I'm sorry.

**Comment by vitorgalvao** [comment]
You could just ask for help doing it.

Will be fixing later.

**Pull request closed without merging**